sceptre/scipool/config/develop/sc-tag-options.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - "develop/sc-portfolio-scheduled-jobs.yaml"
 sceptre_user_data:
   CostCenters: !from_json
-    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true'
+    - !request 'https://mips-api.finops.sageit.org/tags?only_codes=101300,101400,101600,112501,119400,121700,120100,122000,122100,122300,122500,312000,314900,401900,506800,613500,990100,990300,990400&enable_other_code=true'
```

Comment:
i'm not sure i like this idea because i think enumerating every code we want on the list is difficult to maintain which is really no better than a static list.  what if one of these codes becomes inactive?

Reply:
@zaro0508 Can you answer @jesusaurus 's question?  I think you are saying that you'd prefer the list to be maintained somewhere else.  Can you suggest an alternative?

Reply:
I've re-worked the underlying PR to be a priority list